### PR TITLE
Launchpad icon added

### PR DESCRIPTION
**Pull request by threezool**
A launchpad icon has now been added for the Website app in regards to issue #100

### CODE REVIEW

File: src/Website/Api/MappingHandlers.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Starcounter;
```
Comment (warpech) [inline src/Website/Api/MappingHandlers.cs]: Please remove unused usings.

File: src/Website/Api/OntologyMap.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+using System.Threading.Tasks;
+using Starcounter;
+
+namespace Website.Api
+{
+    internal class OntologyMap
+    {
+        public void Register()
+        {
+            UriMapping.Map("/website/app-name", UriMapping.MappingUriPrefix + "/app-name");
```
Comment (warpech) [inline src/Website/Api/OntologyMap.cs]: This is a obsolete API. Please use `Blender.MapUrl`. See People app `develop` branch for reference.

File: src/Website/wwwroot/Website/viewmodels/AppIcon.html [hunk 3]
```diff
@@ -0,0 +1,3 @@
+﻿<template>
+    <img slot="Website/website-image" src="/Website/images/website.png" style="width: 100%; height: 100%" />
```
Comment (warpech) [inline src/Website/wwwroot/Website/viewmodels/AppIcon.html]: Why not `slot="Website/app-icon"`? Would be a little bit more informative

File: src/Website/Api/OntologyMap.cs (hunk 2 above)
Comment (threezool) [inline src/Website/Api/OntologyMap.cs]: After i changed from 
`UriMapping.Map("/website/app-name", UriMapping.MappingUriPrefix + "/app-name");`
to 
`Blender.MapUri("/website/app-name", "/app-name");` (also tried keeping the MappingUriPrefix)
it breaks and the icon will not show, am i missing something?

Comment (warpech) [inline src/Website/Api/OntologyMap.cs]: Try `Blender.MapUri("/website/app-name", "app-name");`

Comment (warpech) [inline src/Website/Api/OntologyMap.cs]: Are you using Launchpad develop branch?

Comment (threezool) [inline src/Website/Api/OntologyMap.cs]: Changed to Dev and also removed the slash but no change. Could it be the Starcounter version that is to low?

The version i have here at the office is Starcounter-RC-2.3.0.6256.

Comment (threezool) [inline src/Website/Api/OntologyMap.cs]: Might have been that i forgot to build after swapping to the Dev branch or the order i started the apps. Now it works again. =)

**Pull request merged**